### PR TITLE
dev-python/indexed_gzip: new package

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.50, Repoman-2.3.11

### CODE REVIEW

File: dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
+	dev-python/cython[${PYTHON_USEDEP}]
+	test? (
+		dev-python/coverage[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-cov[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild]: Please kill coverage testing if it's obligatory, and ignore it if it's optional. It's not valuable for ebuilds. Then you should be able to go for py3.7.

File: dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+src_compile() {
+	if use test; then
+		export INDEXED_GZIP_TESTING=1
+	fi
+	distutils-r1_src_compile
+}
+
+python_test() {
+	cp conftest.py "${BUILD_DIR}"
```
Comment (mgorny) [inline dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild]: ||die

File: dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	cd "${BUILD_DIR}" || die
+	pytest -v -s || die
```
Comment (mgorny) [inline dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild]: `-vv`, without `-s`. We really want verbose.

**Comment by mgorny** [comment]
(also signoff)

**Comment by TheChymera** [comment]
@mgorny done

File: dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
+	dev-python/cython[${PYTHON_USEDEP}]
+	test? (
+		dev-python/coverage[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline dev-python/indexed_gzip/indexed_gzip-0.8.7.ebuild]: Soooo… is this still used?

**Comment by mgorny** [comment]
Thanks. I'm going to test it, also with py3.7, and merge if it passes.

**Comment by mgorny** [comment]
@TheChymera, hmm, how long do the tests take? `lib/indexed_gzip/tests/test_zran.py::test_seek_then_read_block` seems to be running for 7 minutes now and I'm wondering whether it's expected or it hangs.

**Comment by mgorny** [comment]
Ok, I'm going to take the liberty of reducing `--nelems` and `--niter` to make the tests finish in reasonable time.

**Pull request closed without merging**